var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -52,10 +52,24 @@ class Paraview(CMakePackage, CudaPackage):
             description='Builds a shared version of the library')
     variant('kits', default=True,
             description='Use module kits')
+
+    legacy_cuda_values = ['native', 'fermi', 'kepler', 'maxwell',
+                          'pascal', 'volta', 'turing', 'all', 'none']
+    cuda_values = legacy_cuda_values.extend(CudaPackage.cuda_arch_values)
     variant('cuda_arch', default='native', multi=False,
-            values=('native', 'fermi', 'kepler', 'maxwell',
-                    'pascal', 'volta', 'turing', 'ampere', 'all', 'none'),
+            values=cuda_values,
```

Comment:
Can we just stick to the values prescribed by the base class `CudaPackage` and do the translation to the values the build system expects only within `cmake_args`?